configs/2020.04.3.ini

```diff
@@ -16,6 +16,6 @@ test_data_path=
 
 [model]
 label_binarizer_path=models/label_binarizer.pkl
-model_path=models/cnn-2020.04.3
+# model_path=models/cnn-2020.04.3
```

Comment:
Don't get why you comment this?

Reply:
Good catch, I need to change some of the configs now that I refactored the code as some non productionised models are saveable because they are neither pickable nor contain a save function which is fine since we only care about reproducing the result

Reply:
I am planning to do any changes necessary as part of the DVC transition which will also further improve the reproducibility